service/service_setup.go

```diff
@@ -34,19 +34,16 @@ import (
 	"github.com/pkg/errors"
 )
 
-func (svc *serviceCore) setupLogging() {
-	if svc.log == nil {
-		err := svc.configProvider.Get("logging").PopulateStruct(&svc.logConfig)
-		if err != nil {
-			ulog.Logger(context.Background()).Info("Logging configuration is not provided, setting to default logger", "error", err)
-		}
+var _simpleCtx = context.Background()
```

Comment:
I think its better that we use `ulog.Logger(_simpleCtx)` in service. If we have context based objects in the future, we will have use of _simpleContext.